app/views/games/show.html.erb

```diff
@@ -66,20 +58,33 @@
         var piece = $.parseHTML(
           '<div class="piece" data-id='+ piece.id +'>' + pieceHtml + '</div>');
 
-        $chessboard.find('.square#'+squareId).append(piece);
-        $chessboard.find('.piece').draggable();
+        $('.square#'+squareId).append(piece);
+        $('.piece').draggable();
       }
+    }
 
-      function movePiece(piece) {
-        var coord = piece.closest('.square').attr('id');
-        var newRow = coord[0];
-        var newCol = coord[1];
-
-        $.post(piecesURL + "/" + piece.data("id"), {
-          _method: "PUT",
-          piece: { row: newRow, col: newCol }
-        }).success();
+    $(".square").droppable({
+      drop: function(event, ui) {
+        var coord = $(this).attr('id').split('');
+        var $piece = $(ui.draggable);
+        movePiece($piece, coord);
       }
     });
+
+    // race condition without setTimeout():
+    // The get request may complete before the post request's back-end logic is
+    //   completely executed.
+    // In the worst case, the get request will retrieve pieces before they are
+    //   updated by Piece#move_to!.
```

Comment:
I want to make sure that I understand the current JS/jQ. Specifically, is every piece on the chessboard updated when the $.post succeeds?

That is what appears to happen when I add some console.log lines to the JS/jQ. 

Was there consideration to create a success function that 1) moves the piece to its new square and 2) removes the captured piece at the new square? Part 2 is complicated I think - the code would have to detect a second piece in that square and determine which one to not display. I understand why the entire board is updated for ease.